cmd/consumer/main.go

```diff
@@ -75,25 +89,58 @@ func main() {
 
 	db, err := database.New(ctx, cfg.DatabaseDSN)
 	if err != nil {
-		log.Fatal(errors.Wrap(err, "opening store db connection"))
+		logger.Fatal("failed to create db connection", zap.Error(err))
 	}
 	defer db.Close()
 
 	logger.Info("creating HN client")
-	hn := hn.New()
+	hnClient := hn.New()
+
+	rmq, err := queue.New(cfg.RabbitMQURL, queueName, logger)
```

Comment:
not really a fan of the naming here for hnClient or rmq
would prefer `hackerNewsClient` and `queueClient`